.env.example

```
@@ -0,0 +1,23 @@
+DISABLE_SECURE_COOKIE=true
+
+# File Upload Configuration
+UPLOAD_DIR=/home/user/code/web/cbt-workbook/uploads
+UPLOAD_PATH=/uploads
+EXTERNAL_UPLOAD_URL=https://cdn.example.com # Uncomment in production to use external CDN or NGINX for file serving
+
+# PostgreSQL Connection Variables
+PGHOST=localhost
+PGPORT=5432
+PGUSER=postgres
+PGPASSWORD=your_password
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The PGPASSWORD key should go before the PGPORT key

(UnorderedKey)